app/controllers/versions_controller.rb

```diff
@@ -10,15 +10,17 @@ def open_ui
   end
 
   def close_ui
-    @description = @object.datastreams['versionMetadata'].current_description
-    @tag = @object.datastreams['versionMetadata'].current_tag
-
-    # do some stuff to figure out which part of the version number changed when opening
-    # the item for versioning, so that the form can pre-select the correct significance level
-    @changed_significance = which_significance_changed(get_current_version_tag(@object), get_prior_version_tag(@object))
+    versions = Dor::Services::Client.object(params[:item_id]).versions.inventory
+    previous_version, current_version = versions.sort(&:versionId).last(2)
```

Comment:
Are the versions not sorted by version ID already?

Reply:
This is what the code in dor-services is doing( https://github.com/sul-dlss/dor-services/blob/main/lib/dor/datastreams/version_metadata_ds.rb#L230) , so I just thought I ought to do it here, although they always do seem to be sorted.

Reply:
It's safe enough, nbd. No harm done in making the sort explicit on this side.